### PR TITLE
Change how to become a maintainer

### LINKED ISSUE #93: Discrepancy between government document and actual way of working
### Description
The GOVERNANCE.md document do not reflect the actual way of working with maintainers and CODEOWNERS file. We do not use the CODEOWNERS file to keep track of the repository maintainers, but instead use GitHub's built in functionality of maintainer lists. We need to update the GOVERNANCE.md document to reflect this.

### Motivation
See above.

### Benefits
We get a less confusing process of adding maintainers, and a more clearly defined election process (since only maintainers are allowed to be nominated to the Technical Community.

### PR DESCRIPTION
**Pull request by henning-roos**
### Applicable Issues
Fixes #93

### Description of the Change
The GOVERNANCE.md document do not reflect the actual way of working with maintainers and CODEOWNERS file. We do not use the CODEOWNERS file to keep track of the repository maintainers, but instead use GitHub's built in functionality of maintainer lists. This change reflect this.

### Benefits
We get a less confusing process of adding maintainers, and a more clearly defined election process (since only maintainers are allowed to be nominated to the Technical Community.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Henning Roos henning.roos@ericsson.com

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Contributors are eligible to vote on Technical Committee elections. Contributors
 
 Being contributor is a prerequisite for becoming a maintainer to the projects they are contributing to.
 In order for a contributor to become a maintainer, the individual must be nominated by one of the existing maintainers with a reference to the individual's history of contributions to the project.
-The nomination is done as a pull request to the *CODEOWNERS* file in corresponding project's git repository. The approval must be done on the pull request by project's maintainers.
+The nomination is discussed and voted on in the maintainer discussion channel. If approved, the nominated contributor must be added to the maintainer list.
```
Comment (magnusbaeck) [inline GOVERNANCE.md:36]: Should we say anything about what "approved" means in this context? Unanimous approval from existing maintainers? Or a majority? Or at least two approvals like for PRs?

**Review by magnusbaeck — COMMENTED** [review]
You might want to update the Applicable Issues section from `#93` to `Fixes #93` to have the issue closed automatically when the PR is merged.

File: GOVERNANCE.md (hunk 1 above)
Comment (m-linner-ericsson) [inline GOVERNANCE.md:36]: Is the "Mainterners discussion channel" described anywhere?

Comment (e-backmark-ericsson) [reply GOVERNANCE.md:36]: I've rephrased this text now

Comment (e-backmark-ericsson) [reply GOVERNANCE.md:36]: I've rephrased this text now

File: GOVERNANCE.md [hunk 2]
```diff
@@ -41,14 +41,12 @@ Determination of the complete criteria for contributors is the responsibility of
...
 
-Being contributor is a prerequisite for becoming a maintainer to the projects they are contributing to.
-In order for a contributor to become a maintainer, the individual must be nominated by one of the existing maintainers with a reference to the individual's history of contributions to the project.
-The nomination is done as a pull request to the *CODEOWNERS* file in corresponding project's git repository. The approval must be done on the pull request by project's maintainers.
-
 ### Maintainers
 
 Maintainers are active contributors and participants in the projects they are maintainers of. Maintainers have the rights to accept pull requests.
 
+In order for a contributor to become a maintainer for a certain project, the individual must be nominated by one of the existing maintainers of that project with a reference to the individual's history of contributions to the same project. The nomination is discussed and voted on amongst the maintainers. Unless unanimous approval within the maintainers group can be reached, the Technical Committee can be consulted for advice. If approved, the nominated contributor must be added to the maintainer list.
```
Comment (m-linner-ericsson) [inline GOVERNANCE.md:48]: Have we defined the "maintainer list" somewhere?

Comment (magnusbaeck) [reply GOVERNANCE.md:48]: I don't think so. I'm not sure it needs to be defined, but we could add a sentence to the top paragraph and reference the GH teams. However, I don't know what status you must have to be able to inspect the members of a team, though (or even see what team controls the maintainership of a git). I reckon you must at least be a member of the organization?

Comment (e-backmark-ericsson) [reply GOVERNANCE.md:48]: What about this instead?
```suggestion
In order for a contributor to become a maintainer for a certain project, the individual must be nominated by one of the existing maintainers of that project with a reference to the individual's history of contributions to the same project. The nomination is discussed and voted on amongst the maintainers. Unless unanimous approval within the maintainers group can be reached, the Technical Committee can be consulted for advice. If approved, the nominated contributor must be added to the GitHub team for the maintainers of this project.
```

**Review by m-linner-ericsson — APPROVED** [review]

**Review by magnusbaeck — APPROVED** [review]

**Pull request merged**